integrations/nvidia/pyproject.toml

```diff
@@ -117,7 +117,6 @@ unfixable = [
   # Don't touch unused imports
   "F401",
 ]
-extend-exclude = ["tests", "example"]
```

Comment:
Why are we including tests in the linting? It's more of a hassle than anything else.

Reply:
I think it makes sense to have some linting in tests, at least to keep formatting consistent. If I'm not mistaken all other integration have it enabled for tests too, same for core.

Also I enabled it cause [tests were failing in CI](https://github.com/deepset-ai/haystack-core-integrations/actions/runs/8892387873/job/24416337070#step:6:38) and I didn't catch that with auto formatting.

We could disable checks for tests but I would very much prefer to keep tests formatting tidy at least. I do agree that type checking can go out of the window for tests. This just enables `ruff` though.

Reply:
Okay if we just stick to `black`/formatting and keep `mypy`/`pylint` disabled.